drupalux/seven
# Apply new checkbox and radio styles

## Issue

This PR fixes #15 and fixes #19 .

## Screenshot / UI changes

![01--checked-states--mac-os-x--chrome](https://user-images.githubusercontent.com/9197058/49236544-9b714500-f3fc-11e8-9d27-c5794ca7ee02.png)
![02--checkbox-error-states--mac-os-x--chrome](https://user-images.githubusercontent.com/9197058/49236561-a0ce8f80-f3fc-11e8-8683-bd140928444f.png)

See https://github.com/drupalux/claro/pull/63#issuecomment-446619227

## Testing instructions

* Add `https://github.com/zolhorvath/seven-refresh/` as `drupal-profile` (`sevenrefresh`) to a Drupal project.
* Remove `core/themes/seven`
* Make sure that this theme is installed to `themes/custom/seven`
* Symlink `core/misc` to `themes/custom/misc`
* Run `yarn && yarn build:css` in the theme's root
* Install a Drupal with profile `sevenrefresh`. This will enable additional field types such as `telephone` or `date_range` and enable `styleguide` Drupal modules.

After that Drupal is installed, Styleguide will be the front page and you'll have the test form at `/contact/checkbox_radio`.
